assignmentOperators.md

```diff
@@ -0,0 +1,55 @@
+### JavaScript Assignment Operators
+---
+Assignment operators assign values to JavaScript variables.
+
+| Operator | Example | Same As |
+|----------|---------|---------|
+| = | x = y | x = y |
+| += | x += y | x = x + y |
+| -= | x -= y |	x = x - y |
+| *= | x *= y |	x = x * y |
+| /= | x /= y | x = x / y |
+| %= | x %= y |	x = x % y |
+| &= | x &= y | x = x & y |
+| ^= | x ^= y | x = x ^ y |
+| |= | x |= y | x = x | y |
+
+---
+
+Examples
+
+The `=` assignment operator assigns a value to a variable.
+```
+var x = 10;
+console.log(x) //10
+```
+The `+=` assignment operator adds a value to a variable.
```

Comment:
I think It is better to describe it like:
The `+=` assignment operator adds the value of the variable and value given in the RHS and it assigns the result to the variable. Is it looking good?

Reply:
Addressed this